web/themes/custom/drevops/components/00-base/reveal/reveal.scss

```
@@ -0,0 +1,47 @@
+//
+// Scroll-triggered fade-up with staggered delay utilities. The 'visible'
+// class is added by reveal.js once the element scrolls into view.
+//
+// The hidden starting state and the transition are applied only when the
+// visitor has not requested reduced motion, so reduced-motion visitors see
+// content in its final position with no entrance animation.
+//
+
+@media (prefers-reduced-motion: no-preference) {
+  .component-reveal {
+    opacity: 0%;
```

Stylelint (17.13.0)

[error] Expected "0%" to be "0" (alpha-value-notation)

(alpha-value-notation)